src/components/Auth/Logout.react.js

```diff
@@ -16,24 +19,31 @@ const deleteCookie = function(name, options = {}) {
 };
 
 const Logout = props => {
-  // Clear cookies
   deleteCookie('loggedIn', { domain: cookieDomain, path: '/' });
   deleteCookie('serverUrl', { domain: cookieDomain, path: '/' });
   deleteCookie('emailId', { domain: cookieDomain, path: '/' });
   deleteCookie('username', { domain: cookieDomain, path: '/' });
-  deleteCookie('showAdmin', { domain: cookieDomain, path: '/' });
+  deleteCookie('isAdmin', { domain: cookieDomain, path: '/' });
   deleteCookie('uuid', { domain: cookieDomain, path: '/' });
 
-  // Redirect to landing page
   if (props.history) {
+    props.actions.logout();
     props.history.push('/');
```

Comment:
Didn't have to call the **fetchMessage** action because of this statement, as this out unmount the ChatApp and remount it, thus call the **fetchMessage** action from ChatApp's **componentDidMount** lifecycle hook.